server/src/main/scala/com/cloudera/livy/server/LivyServer.scala

```diff
@@ -48,9 +48,11 @@ class LivyServer extends Logging {
 
   private var server: WebServer = _
   private var _serverUrl: Option[String] = None
+  // make livyConf accessible for testing
+  var livyConf: LivyConf = _
```

Comment:
This probably could be done more cleanly, but at the very least, `private[livy]`.